packages/xo-web/src/xo-app/hub/index.js

```diff
@@ -0,0 +1,76 @@
+import _ from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import { addSubscriptions } from 'utils'
+import { Container, Col, Row } from 'grid'
+import { injectState, provideState } from 'reaclette'
+import { isEmpty, map, mapValues, orderBy } from 'lodash'
+import { subscribeResourceCatalog } from 'xo'
+
+import Page from '../page'
+import Resource from './resource'
+
+// ==================================================================
+
+const DEFAULT_SORT_OPTION = {
+  labelId: 'hubSortByName',
+  sortBy: 'name',
+  sortOrder: 'asc',
+}
+
+const HEADER = (
+  <Container>
+    <h2>
+      <Icon icon='menu-hub' /> {_('hubPage')}
+    </h2>
+  </Container>
+)
+
+export default decorate([
+  addSubscriptions({
+    catalog: subscribeResourceCatalog({ filters: { hub: true } }),
+  }),
+  provideState({
+    initialState: () => ({
+      sortBy: DEFAULT_SORT_OPTION.sortBy,
+      sortOrder: DEFAULT_SORT_OPTION.sortOrder,
+    }),
+    computed: {
+      resources: ({ availableResources, sortBy, sortOrder }) =>
+        orderBy(availableResources, res => res[sortBy], sortOrder),
+      availableResources: (_, { catalog }) => {
+        if (catalog !== undefined) {
+          delete catalog._namespaces
+        }
+        return map(mapValues(catalog, 'xva'), (info, namespace) => ({
+          ...info,
+          namespace,
+        }))
+      },
+    },
+  }),
+  injectState,
+  ({ state: { resources } }) => (
+    <Page header={HEADER} title='hubPage' formatTitle>
+      <Row>
+        {isEmpty(resources) ? (
+          <h2 className='text-muted'>
+            &nbsp; {_('hubVmNoAvailableMsg')}
+            <Icon icon='alarm' color='yellow' />
+          </h2>
+        ) : (
+          map(resources, (info, namespace) => (
+            <Col key={namespace} mediumSize={3}>
+              <Resource
+                className='card-style'
```

Comment:
You can use a [`Card`](https://github.com/vatesfr/xen-orchestra/blob/8b454f0d39ca26a17b7faf417cb67d17c625d4fe/packages/xo-web/src/common/card.js#L18).